src/commands/create.c

```diff
@@ -22,6 +22,10 @@
 
 extern struct start_data start_data;
 
+/* ignore -pedantic to cast handle_option_console, a function pointer, to a
+ * void* */
+#pragma GCC diagnostic push
```

Comment:
I'd like to hear @jodh-intel opinion on the pragmas. Great that you've nullified the warnings, but in-code GCC specific pragmas are not generally to my liking :-)  If James is OK, then I think we'll probably be fine.